x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -554,7 +554,7 @@ export const deletePackageHandler: FleetRequestHandler<
       pkgName,
       pkgVersion,
       esClient,
-      force: request.body?.force,
+      force: request.query?.force,
```

Comment:
Should we support both `request.query?.force`  and `request.body?.force` before we remove the deprecated API?

Reply:
using `request.body.force` was always undefined, that's why the force flag didn't work
tested like this:
```
DELETE kbn:/api/fleet/epm/packages/system/1.38.2
{
  "force": true
}
```